code/unclassified/gcd_lcm

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+using namespace std;
+
+
+
+int gcd(int n, int m){
+    //find the gcd of given two numbers
+    int q=0,r=0;
+    max=find_max(n,m);
```

Comment:
variable `max` has not been defined and function `find_max()` should be above this function. Similar issues exists in other parts as well. Kindly correct to get this merged 👍

Reply:
thanks